tests/factory_fixtures/property.py

```diff
@@ -12,7 +12,7 @@ def _property_attributes(archived=False, manager_ids=None):
             "name": faker.unique.name(),
             "address": faker.address(),
             "city": faker.city(),
-            "unit": faker.building_number(),
+            "num_units": faker.building_number(),
```

Comment:
So `faker.building_number()` actually returns a string of a number, but since Marshmallow tries to convert it to an int during deserialization, this works.  

That said, I'd prefer to pass in an actual integer just to be explicit and replicate the actual data as closely as possible.
I believe `faker.building_number()` was probably only used due to the confusion about the "unit" name before. 

I'd suggest something like this:
```suggestion
            "num_units": faker.random_int(min=1),
```